op-chain-ops/cmd/receipt-reference-builder/print.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/urfave/cli/v2"
+)
+
+var printCommand = &cli.Command{
+	Name:   "print",
+	Usage:  "read an aggregate file and print it to stdout",
+	Flags:  []cli.Flag{FilesFlag, InputFormatFlag},
+	Action: print,
+}
+
+func print(ctx *cli.Context) error {
+	log := log.New()
+	files := ctx.StringSlice("files")
+	r := formats[ctx.String("input-format")]
+	for _, f := range files {
+		a, err := r.readAggregate(f)
+		if err != nil {
+			log.Error("failed to read aggregate", "file", f, "err", err)
+			return err
+		}
+		log.Info("aggregate", "aggregate", a)
```

Comment:
How is the output here? It might be nice to do a raw fmt.Println here (or do some formatting on top without the extra logging stuff)

Reply:
It's noisy for sure, but I'm not sure extra formatting would make it better. The aggregate structure is just a massive uint:uint map, and `print` is just meant as a sanity check that the structure is essentially well formed.

I'll take a look at better print formats

Reply:
If it's a sanity check, we can leave as is